packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -4650,6 +4659,16 @@ ${associatedNodes.join(", ")}`,
 			);
 		}
 
+		// Don't let two firmware updates happen in parallel
+		if (this.isAnyOTAFirmwareUpdateInProgress()) {
```

Comment:
I'm sure this change will get complaints from users who queue a bunch of firmware updates for sleeping nodes. However I think it makes sense, at least until I can implement some kind of task scheduling in the driver.